shared/data-models/src/models/User.ts

```diff
@@ -267,6 +273,20 @@ export default class UserModel {
     }
   }
 
+  logUserAction = async (
+    userId: number,
+    serverId: number,
+    action: UserAction,
+    details: string
+  ) =>
+    await this.query(
```

Comment:
No need to async/await this, just return the promise from query

Reply:
True

Reply:
True